README.md

```diff
@@ -30,6 +30,9 @@ Install this addon via ember-cli:
 ember install ember-can
 ```
 
+After install add `import 'ember-can/resolver';` inside your app.[js|ts]
```

Comment:
Is there not a way we can make this automatic?

Reply:
Yes over blueprint for new installations.... i will look to do it asap

Reply:
For current installs we should add under changeloga that this change must be applied

Reply:
Sorry, I didn't mean automatically add this to files, I meant remove the need for it completely. Like using the initializer before or whatever.

Reply:
I'm getting `No ability type found for foo` for all my abilities in my app. Will this PR fix that? I definitely think we should have some kind of initializer that sets all this up automatically. I don't think initializers are deprecated or anything, so I am not sure why we removed it?

Reply:
@RobbieTheWagner yes this is exact the error which i have also got in our app... this PR solves the error. There was no test present which has tested this yet (with this PR we have one).

The initializer has solved the error in which they were run years ago... but they have exported always a empty initializer and added this code part outside of them. It was more/less a workaround to don't teach consumers that they need to add something inside the app.
At the end the previouse looks for me more as a "hack".

I have already tried to readd it like before (inside src folder and also directly exported like blueprints) but it isn't working...